### DISCUSSION TITLE
Option to Show Cursor during Gameplay for Android

### DISCUSSION THREAD

**Discussion by aoi-chan-baka** [ppy/osu#14427] [Ideas] [answered]
Idea: Add an option to Show Cursor during Gameplay for Android users.

It's really hard to play lazer on android with drag or a stylus when playing because i can't see the cursor on screen, practicing to click (key z, x) makes it harder too because i can't see the cursor's timing going to the hit circle. Adding an option to Show/Hide Cursor During Gameplay makes things easier for some of us android users. :)

**Comment by peppy** [comment, marked as answer]
Is this because the cursor lags behind your pen/finger?

**Reply by aoi-chan-baka** [reply]
No, the cursor like brings down its brightness to like 3% when i play a beatmap and i can't see it.

**Reply by aoi-chan-baka** [reply]
![Screenshot_2021-08-21-14-05-39-113_sh ppy osulazer](https://user-images.githubusercontent.com/88275597/130312380-e69ab32d-194f-474b-9d18-20ef017fc8a3.jpg)

I turned on Relax because it fails too fast for me to screenshot

As you can see, the less visible yellow dot is there (in the middle) and when i try to do DT maps, i fail because i can't see the cursor

**Reply by aoi-chan-baka** [reply]
The cursor skin is 100% visible, the skin didn't make the cursor go less visible

Edit: if the cursor is made that way, it's okay. But adding an option to show/hide the cursor might be cool

**Reply by frenzibyte** [reply]
That's not intentional, no. The cursor is supposed to be hidden when touch input is received.

**Reply by aoi-chan-baka** [reply]
@frenzibyte oh, didn't know that, but it feels uncomfortable to play without a cursor though

**Reply by frenzibyte** [reply]
Seeing a cursor in the main menu or anywhere else feels wrong on a touch device, the fact that you can't consider your stylus as the cursor in game raises the originally asked question:
 > Is this because the cursor lags behind your pen/finger?

**Reply by aoi-chan-baka** [reply]
Makes sense now… well i guess i'll play it like that for now until i get a pc and play osu!std

**Reply by peppy** [reply]
> Makes sense now… well i guess i'll play it like that for now until i get a pc and play osu!std

can you explain why you bleep the need to see a cursor? that's not how any other touch game works so i am always curious about requests like this.

**Reply by aoi-chan-baka** [reply]
@peppy i was referring to the same option that osu!droid had about the cursor… and i just messed up on the cursor visibility bit where it really needs to be shown so i'm sorry about that :)
